abstract.tex

```diff
@@ -1,13 +1,19 @@
 \begin{abstract}
         The French 2012-2015 Commission Nationale d'Evaluation Reports
-\cite{cne2_reports_2015} emphasize preparation for a transition from \glspl{LWR} to \glspl{SFR}.
```

Comment:
It's a style rule: citations aren't appropriate in abstracts because an abstract must be able to stand completely alone. If you'd like to cite these reports, it should be done in  the introduction. The abstract, also, technically, shouldn't have any acronyms either, but that's less of a big deal.

Reply:
I have added this citation to the introduction.

Reply:
Cool thanks!